src/crl.rs

```diff
@@ -135,8 +137,24 @@ impl<'a> TryFrom<&'a [u8]> for CertRevocationList<'a> {
     ///
     /// [^1]: <https://www.rfc-editor.org/rfc/rfc5280#section-5>
     fn try_from(crl_der: &'a [u8]) -> Result<Self, Self::Error> {
+        CertRevocationList::new(crl_der, der::TWO_BYTE_DER_SIZE)
+    }
+}
+
+impl<'a> CertRevocationList<'a> {
+    /// Try to parse the given bytes as a RFC 5280[^1] profile Certificate Revocation List (CRL),
+    /// allowing a CRL of up to `size_limit` bytes.
+    ///
+    /// Webpki does not support:
+    ///   * CRL versions other than version 2.
+    ///   * CRLs missing the next update field.
+    ///   * CRLs missing certificate revocation list extensions.
+    ///   * Delta CRLs.
+    ///
+    /// [^1]: <https://www.rfc-editor.org/rfc/rfc5280#section-5>
+    pub fn new(crl_der: &'a [u8], size_limit: usize) -> Result<Self, Error> {
```

Comment:
I'm not sure this is useful if `try_from` becomes relaxed about seeing large CRLs. A caller that wants to bound the size of their buffer needs to be doing it before the storage for `crl_der` is allocated.